app/views/content/life-as-a-teacher/change-careers/support-with-changing-careers/_article.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <p>If you're thinking about changing careers to become a teacher, knowing where to start and understanding what your options are can seem overwhelming. There's lots of dedicated support available to help you navigate the change.</p>
+    <h2 class="heading--box-blue">Get advice and guidance from an expert</h2>
+    <h3>Teacher training advisers</h3>
+    <p>Our experienced <a href="/teacher-training-advisers">teacher training advisers</a> offer free one-to-one support to thousands of people each year to support them through a career change into teaching. All of our advisers have years of teaching experience, so can also give practical advise and insights on what life in the classroom is like.</p>
+    <p>If you have reservations about becoming a teacher, they may know of support available that you're not aware of, or could give practical advice on how other career changers have tackled barriers to becoming a teacher.</p>
+    <p>You can chat to your dedicated adviser by phone, text or email.</p>
+    <p>Find out more about <a href="/teacher-training-adviser/sign_up/identity">getting a teacher training adviser</a>.</p>
+    <h3>Now Teach</h3>
+    <p>You may be able to get <a href="https://nowteach.org.uk/">support from the charity Now Teach</a>. They specifically help people get into teaching who are changing career.</p>
+  </section>
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "Changing careers might seem daunting, but it will be the best decision you ever make. Teaching is such a rewarding career and offers levels of satisfaction that other professions could only dream of providing",
+    name: "Phil, career changer",
+    large: false
+  ) %>
+  </section>
+  <section class="col col-720">
+    <h2 class="heading--box-blue">Applying for teacher training</h2>
+    <p>When you apply for teacher training, you'll need to include things like proof of qualifications, and a written personal statement explaining why you want to become a teacher.<p>
+    <p>To help show your qualifications, you can <a href="https://www.gov.uk/replacement-exam-certificate">get a certified statement of your exam results</a> if you did your GCSEs and A levels in the UK. You should also be able to request a copy of your degree certificate from the organisation where you studied in the UK.</p>
+    <p>For some career changers, it may have been a long time since they applied for a training course or wrote things like personal statements.</p>
```

Comment:
This tone sounds a bit different to the rest of it...maybe stick to referring to them as 'you' so could be something like
If it's been a while since you've applied for a training course, you can get one-to-one guidance from an adviser. An adviser can help you with writing your personal statement, you can also explore tips and guidance on etc